src/controllers/adminController.js

```diff
@@ -3,9 +3,41 @@ const Album = require('../models/Album');
 const adminHomePage = async (req, res) => {
   const allAlbums = await Album.find({}).sort({ price: 1 }); // Fetch all albums from the database and sort them by price in ascending order
   res.render('adminPage', { albums: allAlbums });
+  /*
+  #swagger.tags = ['Admin']
+  #swagger.summary = 'Admin Home Page'
+  #swagger.description = 'Fetch and render the admin home page.'
+  #swagger.responses[200] = {
+    description: 'Admin home page fetched successfully.',
+    content: { 'text/html': {} }
+  }
+*/
 };
 
-const { updatePriceOfAlbums } = require('../controllers/albumController');
+const {
+  updatePriceOfAlbums: originalUpdatePrice,
+} = require('../controllers/albumController');
+const updatePriceOfAlbums = async (req, res) => {
+  /*#swagger.summary = 'Update prices of albums passed in req.body'
```

Comment:
We can add a tag here so that this shows up together with the other admin endpoint:

```suggestion
  /*#swagger.summary = 'Update prices of albums passed in req.body'
    #swagger.tags = ['Admin']
```